sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformation.scala

```diff
@@ -58,11 +58,12 @@ case class ScriptTransformation(
     child.execute().mapPartitions { iter =>
       val cmd = List("/bin/bash", "-c", script)
       val builder = new ProcessBuilder(cmd)
+      builder.redirectErrorStream(true)
```

Comment:
There's already a logic to consuming the stdout which would assemble it to be the sql output. 
So I want to reuse the same logic for stderr here. 
This issue is reproducible and can be cover by the unit test .
@srowen could you please elaborate your concern for the "not expected" behavior ?